src/main/java/com/doit/api/config/WebSecurityConfig.java

```diff
@@ -12,18 +15,22 @@
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
+@Configuration
 @EnableWebSecurity
+@AllArgsConstructor
 class WebSecurityConfig extends WebSecurityConfigurerAdapter {
 
     @Autowired
-    private UserDetailsLoader userDetailsLoader;
+    private final UserService userService;
```

Comment:
Please use Constructor DI instead of `@Autowired`.

Reply:
The same thing goes for all DI sections.

Reply:
Done.